11:15 <@sberg> _rene_, sure, I know what's going on; just hold on for a sec
11:15 <@_rene_> cool
11:16 -!- aruiz [aruiz@nat/redhat/x-wcewybdbkmlrugzi] has joined 
          #libreoffice-dev
11:31 <@sberg> _rene_, and what does the versionrc's buildid= line read in the 
               resulting .deb you install?  not sure how exactly we e.g. for 
               Fedora manage to get some odd git sha into that, but your 
               problem is that an empty buildid= line leads to a zero-byte 
               .../user/extensions/buildid, and the odd osl_readLine 
               implementation responds to that with osl_File_E_AGAIN instead of 
               osl_File_E_None and an empty line
11:31 <@_rene_> depends on whether git or non-git
11:32 <@sberg> ...and then the logic in LO keeps thinking it needs to restart 
               soffice.bin after extension syncing, so you're caught in a 
               livelock
[...]
11:40 <@sberg> _rene_, ok
11:41 <@sberg> _rene_, quickfix for you would be to change the line towards the 
               end of instsetoo_native/CustomTarget_setup.mk,
11:41 <@sberg> && echo 'buildid=$(shell cd $(SRCDIR) && git log -1 
               --format=%H)' \
11:41 <@sberg> to instead use a fixed buildid=fake say
11:41 <@_rene_> or to use the same thing as make install does, as it adda a 
                proper value there
11:41 <@sberg> that should only affect the instdir/program/versionrc used for 
               subsequentchecks, not the one that ends up in .debs
11:42 <@_rene_> but yeah, ok
11:42 < loircbot> LibreOffice (core) [libreoffice-4-2] tomaz.vajngerl * 
                  svgio/source/svgreader/ (svgstyleattributes.cxx 
                  svgsvgnode.cxx): fdo#74743 avoid infinite loop when gathering 
                  "svg" element styles
11:42 <@_rene_> will try
11:42 < IZBot> LibreOffice-Presentation critical/medium NEW Impress 
               systematically freezes when trying to open a file 
               https://bugs.libreoffice.org/show_bug.cgi?id=74743
11:43 <@sberg> _rene_, yep, there's two things that should be addressed, make 
               instsetoo_native/CustomTarget_setup.mk and 
               solenv/bin/modules/installer/scriptitems.pm come up with a 
               consistent buildid value, and fix osl_readLine to not report 
               EAGAIN on a zero-sized file

--- a/instsetoo_native/CustomTarget_setup.mk-old	2014-04-16 11:44:15.055275384 +0200
+++ b/instsetoo_native/CustomTarget_setup.mk	2014-04-16 11:45:00.143276067 +0200
@@ -160,7 +160,7 @@
 		echo '[Version]' \
 		&& echo 'AllLanguages=$(if $(gb_WITH_LANG),$(gb_WITH_LANG),en-US)' \
 		&& echo 'BuildVersion=$(BUILD_VER_STRING)' \
-		&& echo 'buildid=$(shell cd $(SRCDIR) && git log -1 --format=%H)' \
+		&& echo 'buildid=fake' \
 		&& echo 'ExtensionUpdateURL=http://updateexte.libreoffice.org/ExtensionUpdateService/check.Update' \
 		&& echo 'ProductMajor=$(LIBO_VERSION_MAJOR)$(LIBO_VERSION_MINOR)$(LIBO_VERSION_MICRO)' \
 		&& echo 'ProductMinor=$(LIBO_VERSION_PATCH)' \
